myc/ubo-informatie-schema-v1.json

```diff
@@ -0,0 +1,79 @@
+{
+    "@context": [
+      "https://w3id.org/did/v1",
+      "https://w3id.org/security/v2",
+      "https://w3c-ccg.github.io/vc-json-schemas/context/vc-json-schema-v0.0.jsonld"
+    ],
+    "id": "https://sphereon-opensource.github.io/factom-vc-contexts/myc/ubo-informatie-schema-v1.json",
+    "type": "https://w3c-ccg.github.io/vc-json-schemas/schema/credential-schema.json",
+    "modelVersion": "1.0",
+    "name": "UBOInformatieSchema",
+    "authored": "2021-07-09T00:00:00+00:00",
+    "schema": {
+      "$schema": "http://json-schema.org/draft-07/schema#",
+      "description": "UBO Informatie",
+      "type": "object",
+      "properties": {
+        "directbelang": {
```

Comment:
Does the percentage allow for decimals?

Reply:
makes sense that you can have decimals. Changed type to number as these allow for decimals.